### PR TITLE
Implement all Feed UI components

### PR DESCRIPTION
**Pull request by Thesmader**
Related to #71.
- Homepage story and schedule UI implementation.
- Fixed tests for feed viewmodel.
- Implemet MM articles UI and improve test coverage.
- Feed view implemented without announcemens and access level implementation.

### CODE REVIEW

**Review by Chinmay-KB — APPROVED** [review]

File: elaichi/lib/ui/views/feed/feed_viewmodel.dart [hunk 1]
```diff
@@ -18,10 +18,11 @@ class FeedViewModel extends BaseViewModel {
   Either<Failure, List<ScheduleEvent>> get scheduleEvents => _scheduleEvents;
 
   /// Calls the required functions to initialise the view model.
-  void initialise() async {
+  Future<void> initialise() async {
     setBusy(true);
-    // Future.wait([fetchCurrentStories(), getSchedule()]);
+    // await Future.wait([fetchCurrentStories(), getSchedule()]);
     await fetchCurrentStories();
+    await getSchedule();
```
Comment (ankank30) [inline elaichi/lib/ui/views/feed/feed_viewmodel.dart:25]: `Future.wait()` is better than these serial `await` statements. Or we can use isolates if possible.

File: elaichi/test/.test_coverage.dart [hunk 2]
```diff
@@ -0,0 +1,28 @@
+// Auto-generated by test_coverage. Do not edit by hand.
+// Consider adding this file to your .gitignore.
```
Comment (ankank30) [inline elaichi/test/.test_coverage.dart:2]: This file is in .gitignore still pushed

File: elaichi/test/service_tests/api_test.dart [hunk 3]
```diff
@@ -1,112 +1 @@
...
-    final result = await fakeApiService.getUser('test', errorMode: 3);
-    expect(result, isNotNull);
-    try {
-      json.decode(result);
-    } catch (e) {
-      expect(e, isInstanceOf<FormatException>());
-    }
-  });
-}
+void main() {}
```
Comment (ankank30) [inline elaichi/test/service_tests/api_test.dart:1]: Can we delete this file completely?

**Review by ankank30 — APPROVED** [review]

File: elaichi/test/service_tests/api_test.dart (hunk 3 above)
Comment (Thesmader) [reply elaichi/test/service_tests/api_test.dart:1]: Will add tests here in future.

File: elaichi/test/.test_coverage.dart (hunk 2 above)
Comment (Thesmader) [reply elaichi/test/.test_coverage.dart:2]: Deleting it.

File: elaichi/lib/ui/views/feed/feed_viewmodel.dart (hunk 1 above)
Comment (Thesmader) [reply elaichi/lib/ui/views/feed/feed_viewmodel.dart:25]: Noted.

**Pull request merged**